src/python/zquantum/core/interfaces/mock_objects.py

```diff
@@ -56,7 +56,23 @@ def run_circuit_and_measure(self, circuit, **kwargs):
 
     def get_expectation_values(self, circuit, operator, **kwargs):
         n_qubits = len(circuit.qubits)
-        values = [random.random() for i in range(n_qubits)]
+        try:
```

Comment:
It shouldn't be done by "try-except", cause it might lead to erratic behaviour.
It's much better to do it using "if-else" statement and e.g. check for the existence of attribute terms inside the "if".